application/build.xml

```diff
@@ -129,6 +129,7 @@
 
                 <!-- Resources that should be used in the Polish build (images, the polish.css file, etc) -->
                 <resources defaultexcludes="no" excludes="readme.txt,.project">
+                    <root dir="${dir.commcare.cases.resources}" />
```

Comment:
Does this look like the right place to ensure that [cases/resources/casedb_instance_structure.xml](https://github.com/dimagi/commcare/blob/bugs/case-data-instance/cases/resources/casedb_instance_structure.xml) makes it into the j2me build?